CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -22,5 +22,8 @@ public class Employee
         [ForeignKey(nameof(Company))]
         public Guid CompanyId { get; set; }
```

Comment:
Field `salary` is lowercase and uses a public field instead of a property, which violates .NET naming conventions and best practices. Additionally, we should consider making it a `decimal` and defining validation attributes if required.

```suggestion
public decimal Salary { get; set; }
```